devicetypes/smartthings/zigbee-smoke-sensor-generic.src/zigbee-smoke-sensor-generic.groovy

```diff
@@ -0,0 +1,141 @@
+ /*
+  *  Copyright 2018 SmartThings
+  *
+  *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  *  use this file except in compliance with the License. You may obtain a copy
+  *  of the License at:
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  *  Unless required by applicable law or agreed to in writing, software
+  *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  *  License for the specific language governing permissions and limitations
+  *  under the License.
+  *  Author : Fen Mei / f.mei@samsung.com
+  *  Date : 2018-07-06
+  */
+
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition (name: "Zigbee Smoke Sensor Generic", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.sensor.smoke", vid: "generic-smoke") {
```

Comment:
I think we should drop "Generic" from the name as we don't include that in our other generic DTHs